components/styled-components.js

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+
```

Comment:
hmm... i'm a bit confused about your structure, things like the h1 or the h3 should be placed in the styles.js (global), because you use them often for each page (if you want to have more in the future), 
but if you outsource all styles for the ToDo list, you don't need another file, you can just leave these special stylings on the todolist.js